src/frontends/tensorflow/src/utils.cpp

```diff
@@ -62,3 +66,120 @@ ov::op::PadType ov::frontend::tensorflow::convert_conv_tf_padding(const ov::fron
 
     return ov::op::PadType::EXPLICIT;
 }
+
+void fill_explicit_pads_vectors(const ov::frontend::tensorflow::NodeContext& node,
+                                bool is_nhwc,
+                                size_t spatial_dims_num,
+                                const std::vector<int64_t>& tf_explicit_paddings,
+                                ov::CoordinateDiff& pads_begin,
+                                ov::CoordinateDiff& pads_end) {
+    if (spatial_dims_num == 2) {
+        TENSORFLOW_OP_VALIDATION(node,
+                                 tf_explicit_paddings.size() == 8,
+                                 "Conv2D expects 8 padding values for EXPLICIT padding mode.");
+        // prepare pads_begin and pads_end attributes for EXPLICIT padding mode
+        if (is_nhwc) {
+            // For NHWC layout, explicit paddings has the following form:
+            // [0, 0, pad_h1, pad_h2, pad_w1, pad_w2, 0, 0]
+            pads_begin.push_back(tf_explicit_paddings[2]);
```

Comment:
looks like we need to check that pads_begin is empty here or we can implement like this:

pads_begin = {tf_explicit_paddings[2], tf_explicit_paddings[4]};
pads_end = {tf_explicit_paddings[3], tf_explicit_paddings[5]};

And please do the same below

Reply:
I think this is nit.
Now this routine is used from the same function that provides empty `pads_begin`

Reply:
this function doesn't know it :)
this can lead to mistake

Reply:
let me fix this in the next PR due to problems with CI, ok?

Reply:
you can also implement some lambda-helper:
```
auto fullfill_pads = [&](ov::CoordinateDiff& pads, const std::vector<int64_t>& indexes) {
   pads.resize(indexes.size());
   for (int i = 0; i < indexes.size(); ++i) {
       pads[i] = tf_explicit_paddings[indexes[i]];
   }
};
```
and the code will be like this:
fullfill_pads(pads_begin, {2, 4});